### PR TITLE
DOCSP-34133 adding read preference info to connection strings

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

Add info on specifying read preference in connection string 
(read/write concern info added in #210)

## STAGING

https://preview-mongodbltranmdb2.gatsbyjs.io/cluster-sync/DOCSP-34133/reference/mongosync/#read-preference
https://preview-mongodbltranmdb2.gatsbyjs.io/cluster-sync/DOCSP-34133/connecting/atlas-to-atlas/#connection-strings
https://preview-mongodbltranmdb2.gatsbyjs.io/cluster-sync/DOCSP-34133/connecting/onprem-to-onprem/#connection-strings
https://preview-mongodbltranmdb2.gatsbyjs.io/cluster-sync/DOCSP-34133/connecting/onprem-to-atlas/#connection-strings

## JIRA

https://jira.mongodb.org/browse/DOCSP-34133

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=656a4bb47a5cb5fef500a0d4

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/reference/mongosync.txt [hunk 1]
```diff
@@ -255,6 +255,12 @@ the destination cluster, ``mongosync`` sets the write concern level to
 For more information on read and write concern configuration and
 behavior, see :ref:`read-concern` and :ref:`write-concern`. 
 
+Read Preference
+~~~~~~~~~~~~~~~
+By default, ``mongosync`` sets the source cluster read preference to
```
Comment (ianf-mongodb) [inline source/reference/mongosync.txt:260]: **Nit** Add a line break after the header.

```suggestion

By default, ``mongosync`` sets the source cluster read preference to
```

File: source/reference/mongosync.txt [hunk 2]
```diff
@@ -255,6 +255,12 @@ the destination cluster, ``mongosync`` sets the write concern level to
...
+:readmode:`nearest` to distribute reads evenly across nodes. To change
+the read preference of clusters, specify the read preference mode
```
Comment (ianf-mongodb) [inline source/reference/mongosync.txt:262]: **Suggestion**

You might want to add some content here on how the user can set the read preference mode.
Also you may want to be explicit and mention "source cluster" similar to `source/includes/read-preference-connection-string.rst`.

Something like:

To change the read preference of the source cluster, specify the read preference mode in the URI of the connection string. For details, see :ref:`connections-read-preference`.

```suggestion
the read preference of clusters, specify the read preference mode.
```

**Review by ianf-mongodb — APPROVED** [review]
Hi @ltran-mdb2, this LGTM with a couple of suggestions. Thanks!

**Review by suspicious-mango — APPROVED** [review]
LGTM.

**Pull request merged**